app/src/components/common/Toast.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div
+    v-if="show"
+    class="toast-container"
+    :class="styleType"
+    :style="{ animationDuration: `${timeout / 1000}s` }"
+  >
+    <div class="icon">
+      <font-awesome-icon
+        :icon="icons[styleType]"
```

Comment:
same here. It is not very obvious what is happening here. It's better to do something like `:icon="getIcon"` and move the logic to `<script>`

Reply:
Thanks for advice, tried to fix it in the recent commit.